helm/microgateway/README.md

```diff
@@ -1,145 +1,293 @@
-# Helm Chart for deployment of WSO2 API Microgateway 
+# Helm Chart for deployment of WSO2 API Microgateway
+
+  
 
 ## Contents
 
-* [Prerequisites](#prerequisites)
-* [Quick Start Guide](#quick-start-guide)
+  
+
+*  [Prerequisites](#prerequisites)
+
+*  [Quick Start Guide](#quick-start-guide)
+
+  
 
 ## Prerequisites
 
+  
+
 * In order to use WSO2 Helm resources, you need an active WSO2 subscription. If you do not possess an active WSO2
-  subscription already, you can sign up for a WSO2 Free Trial Subscription from [here](https://wso2.com/free-trial-subscription)
-  . Otherwise you can proceed with docker images which are created using GA releases.<br><br>
+
+subscription already, you can sign up for a WSO2 Free Trial Subscription from [here](https://wso2.com/free-trial-subscription)
+
+. Otherwise you can proceed with docker images which are created using GA releases.<br><br>
+
+  
 
 * Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
-(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (compatible with v1.10) in order to run the 
+
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (compatible with v1.10) in order to run the
+
 steps provided in the following quick start guide.<br><br>
 
+  
+
 * An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup/pick-right-solution/).<br><br>
 
+  
+
 * Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). Please note that Helm resources for WSO2 product
+
 deployment patterns are compatible with NGINX Ingress Controller Git release [`nginx-0.22.0`](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.22.0).
-  
+
 ## Quick Start Guide
+
 >In the context of this document, <br>
->* `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-microgateway`](https://github.com/wso2/kubernetes-microgateway/)
+
+>*  `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-microgateway`](https://github.com/wso2/kubernetes-microgateway/)
+
 Git repository. <br>
->* `HELM_HOME` will refer to `<KUBERNETES_HOME>/helm/microgateway`. <br>
+
+>*  `HELM_HOME` will refer to `<KUBERNETES_HOME>/helm/microgateway`. <br>
+
+  
 
 ##### 1. Clone the Kubernetes Resources for WSO2 Identity Server Git repository.
 
+  
+
 ```
+
 git clone https://github.com/wso2/kubernetes-microgateway.git
+
 ```
 
+  
+
 ##### 2. Provide configurations.
 
+  
+
 a. The default product configurations are available at `<HELM_HOME>/confs` folder. Change the
+
 configurations as necessary.
 
+  
+
 b. Open the `<HELM_HOME>/values.yaml` and provide the following values.
 
+  
+
 ###### WSO2 Subscription Configurations
```

Comment:
@chathuraswso2 please check the formatting of the configurations and the table.
![Screenshot from 2020-09-15 11-25-37](https://user-images.githubusercontent.com/9444555/93171503-d7e37180-f746-11ea-8d66-6dc8ef233e30.png)

Reply:
Fixed in https://github.com/wso2/kubernetes-microgateway/pull/21/commits/5a7cf5fe5f9fdbad72a8d58b2971310548a0dc4d commit